09-intermediate-rails/api-creating-items.md

```diff
@@ -221,6 +226,37 @@ describe "index" do
       expect(body).must_equal []
     end
   end
+
+  describe "create" do
+    # ... let(:pet_data) ...
+
+    it "can create a new pet" do
+      expect {
+        post pets_path, params: pet_data
+      }.must_differ "Pet.count", 1
+
+      check_response(expected_type: Hash, expected_status: :created)
+    end
+
+    it "will respond with bad_request for invalid data" do
+      # Arrange - using let from above
+      # Our PetsController test should just test generically
+      # for any kind of invalid data, so we will randomly pick
+      # the age attribute to invalidate
+      pet_data[:pet][:age] = nil
+
+      expect {
+        # Act
+        post pets_path, params: pet_data
+
+      # Assert
+      }.wont_change "Pet.count"
+
+      body = check_response(expected_type: Hash, expected_status: :bad_request)
+      expect(body["errors"].keys).must_include "age"
+    end
+
+  end
```

Comment:
line 192 is written to imply the code snippet is only showing the `index` tests. 
Can we either change line 192 or remove the `show` tests from this snippet?
I think I have a slight preference to keep it to just the `index` tests because I think less code will help reduce distraction and make the message more clear.